src/Components/Onboarding/Components/OnboardingGene.tsx

```diff
@@ -51,7 +53,14 @@ const OnboardingGene: FC<OnboardingGeneProps> = ({ gene, description }) => {
           {artworks.map(artwork => {
             return (
               <Fragment key={artwork.internalID}>
-                <ArtworkGridItemFragmentContainer artwork={artwork} />
+                <ArtworkGridItemFragmentContainer
+                  artwork={artwork}
+                  onMouseUp={() => {
+                    setTimeout(() => {
+                      onDone()
+                    }, 10)
```

Comment:
Need this slight delay because there's a race condition between clicking an artwork + transition to new page and the close. Can't override one so we piggy back on another event (`onMouseUp` - also works on mobile) and perform an action after the default action executes (navigate to artwork).

Reply:
does this also handle the issue of seeing the "welcome to artsy" screen weirdly pop-up before the user is navigated to the artist page??

Reply:
Yup that's exactly what this does